modules/src/ics02_client/height.rs

```diff
@@ -133,12 +155,19 @@ impl std::fmt::Debug for Height {
 }
 
 /// Custom debug output to omit the packet data
-impl std::fmt::Display for Height {
+impl Display for Height {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
         write!(f, "{}-{}", self.revision_number, self.revision_height)
     }
 }
 
+/// Custom debug output to omit the packet data
```

Comment:
I guess this comment (and on line 157 also) is a copy-paste.

Reply:
Removed.